provisioning/roles/docker/files/docker-compose.yml

```diff
@@ -0,0 +1,171 @@
+version: "3"
+
+services:
+  elasticsearch:
+    image: elasticsearch:5.6.10
+    container_name: elasticsearch
+    restart: always
+    environment:
+      - "bootstrap.memory_lock=true"
+      - "ES_JAVA_OPTS=-Xms4g -Xmx4g"
+    volumes:
+      - /home/ubuntu/snowplow/elasticsearch/data:/usr/share/elasticsearch/data
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    logging:
+      options:
+        max-size: "50M"
+        max-file: "10"
+    ports:
+      - "9200:9200" 
+
+  kibana:
+    image: kibana:5.6.10
+    container_name: kibana
+    restart: always
+    ports:
+      - "5601:5601"
+    depends_on:
+      - elasticsearch
+
+  elasticsearch-loader-good:
+    image: snowplow-docker-registry.bintray.io/snowplow/elasticsearch-loader:0.10.1
+    container_name: elasticsearch-loader-good
+    command: [ "--config", "/snowplow/config/snowplow-es-loader-good.hocon" ]
+    restart: always
+    depends_on:
+      - elasticsearch
+    volumes:
+      - /home/ubuntu/snowplow/configs:/snowplow/config
+    logging:
+      options:
+        max-size: "1M"
+        max-file: "10"
+    environment:
+      - "SP_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
I wouldn't specify `Xms` for any of them

Reply:
why was this removed altogether, I would still specify `Xmx`

Reply:
Specified only Xmx now.